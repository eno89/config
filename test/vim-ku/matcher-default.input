" Test: vim-ku-matcher-default
function s:test_ku_matcher_default_matches_p()  "{{{1
  Title "ku#matcher#default#matches_p()"

  let args = {'pattern': 'rail'}

  Assert !!ku#matcher#default#matches_p({'word': 'rail'}, args) ==> 1
  Assert !!ku#matcher#default#matches_p({'word': 'a rail'}, args) ==> 1
  Assert !!ku#matcher#default#matches_p({'word': 'rail gun'}, args) ==> 1
  Assert !!ku#matcher#default#matches_p({'word': 'a rail gun'}, args) ==> 1
  Assert !!ku#matcher#default#matches_p({'word': 'arailgun'}, args) ==> 1

  Assert !!ku#matcher#default#matches_p({'word': ''}, args) ==> 0
  Assert !!ku#matcher#default#matches_p({'word': 'reil gan'}, args) ==> 0
  Assert !!ku#matcher#default#matches_p({'word': 'RAILgun'}, args) ==> 0
endfunction




Test  "{{{1

" __END__
" vim: filetype=vim foldmethod=marker
