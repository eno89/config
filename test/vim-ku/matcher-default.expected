==== ku_matcher_default_matches_p 
---- It should return true for matched candidate 
.....
---- It should return false for unmatched candidate 
...



**** Result ****


8 examples, 0 failures
